README.md

```diff
@@ -17,7 +17,12 @@
 - Copy https://github.com/Abel-desu/Math-Magicians.git
 - Get the directory that you want to clone the repository.
 - Open the command prompt in this directory.
+
 - Type git clone https://github.com/Abel-desu/Math-Magicians.git
+
+
+- Type git clone https://github.com/Abel-desu/Math-Magicians.git
+
 - Go to the repository folder in your command prompt cd moviehub
```

Comment:
- [ ] You have repeated instructions here. I would recommend you remove it and keep your documentation neat

- [ ] Additionally, on line 26, I believe we cd into `Math-Magicians`, not `moviehub`. Kindly fix that